backend/tests/prisonerWorkInsidePrisonDetails.test.ts

```diff
@@ -15,34 +15,44 @@ describe('Prisoner work inside prison details controller', () => {
     getDetails: jest.fn(),
   }
   const esweService = {
```

Comment:
Could maybe do with a test for pagination, although an int test is probably more useful for that